README.md

```diff
@@ -7,60 +7,28 @@ Feedstock license: [BSD-3-Clause](https://github.com/conda-forge/napari-feedstoc
 About napari
 ------------
 
-Home: http://napari.org
-
-Package license: BSD-3-Clause
-
-Summary: a fast n-dimensional image viewer in Python
-
-Development: https://github.com/napari/napari
-
-Documentation: http://napari.org
-
-a fast n-dimensional image viewer in Python
-
-About napari
-------------
-
-Home: http://napari.org
+Home: http://napari.org/
 
 Package license: BSD-3-Clause AND MIT
 
-Summary: a fast n-dimensional image viewer in Python
+Summary: a fast n-dimensional image viewer in Python, with only the required dependencies
 
 Development: https://github.com/napari/napari
 
-Documentation: http://napari.org
-
-a fast n-dimensional image viewer in Python
+Documentation: http://napari.org/
 
 About napari-base
 -----------------
 
-Home: http://napari.org
+Home: http://napari.org/
 
 Package license: BSD-3-Clause AND MIT
 
 Summary: a fast n-dimensional image viewer in Python, with only the required dependencies
 
 Development: https://github.com/napari/napari
 
-Documentation: http://napari.org
-
-About napari-menu
```

Comment:
The `napari-menu` package is still defined in `recipe.yaml`

Reply:
So we should fix it?

Reply:
Sure, but not here. Opened https://github.com/conda-forge/conda-smithy/issues/2356